wavefront/resource_dashboard.go

```diff
@@ -369,6 +369,11 @@ func resourceDashboard() *schema.Resource {
 					Required:    true,
 					Description: "Name of the Chart",
 				},
+				"base": {
+					Type:        schema.TypeInt,
+					Optional:    true,
+					Description: "Base of logarithmic scale, 0 for linear",
```

Comment:
`Base of logarithmic scale, default is 0 for linear scale.`

Reply:
Reworded the description so that it is clear that 0 is also the default.